recipes/tabulate/run_test.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+import jupyter_kernel_test
+
+class XCppTests(jupyter_kernel_test.KernelTests):
+
+    kernel_name = 'xcpp17'
```

Comment:
I think we should use 17 here because of [this](https://github.com/p-ranav/tabulate/blob/ff9dc4cdfa5c522acb15e87f8608031e63cdf619/CMakeLists.txt#L22), do we have it?

Reply:
This entire file isn't needed for tabulate. This is for testing a jupyter kernel (but tabulate is unrelated to jupyter :)